helmfile/values/fluentd-user.yaml.gotmpl

```diff
@@ -3,6 +3,10 @@ nodeSelector: {{- toYaml .Values.fluentd.user.nodeSelector | nindent 2  }}
 affinity:     {{- toYaml .Values.fluentd.user.affinity | nindent 2  }}
 tolerations:  {{- toYaml .Values.fluentd.user.tolerations | nindent 2  }}
 
+image:
+  repository: {{ .Values.fluentd.image.repository }}
```

Comment:
These values do not need to be configured from wc-config. So move the values from wc-config to here.